### PR TITLE
Allow for running custom init scripts via volume mounted dir

### PR DESCRIPTION
**Pull request by shanemcd**
Ping @cutwater - here's the idea we chatted about.

### CODE REVIEW

File: docker/entrypoint.sh [hunk 1]
```diff
@@ -30,6 +30,23 @@ install_local_deps() {
     done
 }
 
+process_init_files() {
+    local f
```
Comment (cutwater) [inline docker/entrypoint.sh:34]: Please use more verbose variable naming.

File: docker/entrypoint.sh [hunk 2]
```diff
@@ -30,6 +30,23 @@ install_local_deps() {
...
+    for f; do
+        case "$f" in
+            *.sh)
+                if [ -x "$f" ]; then
```
Comment (cutwater) [inline docker/entrypoint.sh:38]: We're using bash in docker scripts. It's safe and preferred to use bash `[[` operator.

File: docker/entrypoint.sh [hunk 3]
```diff
@@ -30,6 +30,23 @@ install_local_deps() {
...
+                    echo "$0: running $f"
+                    "$f"
+                else
+                    echo "$0: sourcing $f"
+                    . "$f"
```
Comment (cutwater) [inline docker/entrypoint.sh:43]: Please use `source` alias instead of `.`

File: docker/entrypoint.sh [hunk 4]
```diff
@@ -30,6 +30,23 @@ install_local_deps() {
     done
 }
 
+process_init_files() {
+    local f
+    for f; do
+        case "$f" in
+            *.sh)
+                if [ -x "$f" ]; then
+                    echo "$0: running $f"
```
Comment (cutwater) [inline docker/entrypoint.sh:39]: please use `log_message()` function for consistency with the rest of the script.

**Comment by cutwater** [comment]
@shanemcd Thank you for submitting this PR. There are minor issues, but overall looks good.

**Comment by shanemcd** [comment]
@cutwater I believe I've addressed all of your feedback.

**Comment by cutwater** [comment]
@shanemcd Thank you. LGTM.

**Pull request merged**